File: joseffe-barroso/Python/exercise12.py
firstNumber = float(input('Digite o primeiro número: '))
secondNumber = float(input('Digite o segundo número: '))
thirdNumber = float(input('Digite o terceiro número: '))
 
if (firstNumber > secondNumber and firstNumber > thirdNumber): 
    print("O primeiro valor informado é o maior")

if (secondNumber > firstNumber and secondNumber > thirdNumber): 
  print("O segundo valor informado é o maior")

if (thirdNumber > firstNumber and thirdNumber > secondNumber): 
  print("O terceiro valor informado é o maior")

if (firstNumber == secondNumber and firstNumber == thirdNumber):
  print("Os valores são iguais")

File: joseffe-barroso/Java/src/entrada_processamento_saida/Exercise12.java
package entrada_processamento_saida;

import java.util.Scanner;

public class Exercise12 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		double firstNumber, secondNumber, thirdNumber;

		System.out.printf("Digite o primeiro número: ");
		firstNumber = scanner.nextDouble();

		System.out.printf("Digite o segundo número: ");
		secondNumber = scanner.nextDouble();

		System.out.printf("Digite o terceiro número: ");
		thirdNumber = scanner.nextDouble();

		if (firstNumber > secondNumber && firstNumber > thirdNumber) {
			System.out.printf("O primeiro valor informado é o maior");
		}

		if (secondNumber > firstNumber && secondNumber > thirdNumber) {
			System.out.printf("O segundo valor informado é o maior");
		}

		if (thirdNumber > firstNumber && thirdNumber > secondNumber) {
			System.out.printf("O terceiro valor informado é o maior");
		}

		if (firstNumber == secondNumber & firstNumber == thirdNumber) {
			System.out.printf("Os valores são iguais");
		}

		scanner.close();
	}

}
